tests/improved-sanity-test/main.yml

```diff
@@ -210,6 +210,24 @@
       tags:
         - atomic_host_check
 
+    # we remove any subscriptions after the upgrade to verify that
+    # 'rpm-ostree status' with the 'unconfigured-state' field present.
+    # https://bugzilla.redhat.com/show_bug.cgi?id=1421867
+    - role: redhat_unsubscribe
+      unconfigured_state: true
+      when: ansible_distribution == 'RedHat'
+      tags:
+        - redhat_unsubscribe
+
```

Comment:
Yeah it will automatically start. But if it was already running, you won't be testing the path which caused that RHBZ to fail, which happened at rpm-ostree startup. Maybe make it an optional `stop_daemon_first` var to the `atomic_host_status_verify` role? It's too bad we can't mix roles and tasks, though we *are* abusing the ansible semantics here. :)